src/wled2.ts

```diff
@@ -48,13 +48,97 @@ export = (RED: Red): void => {
     });
   });
 
+  function which(val1: any, val2: any, wrapper?: any): any {
+    const val = val1 !== null && val1 !== undefined ? val1 : val2 !== null && val2 !== undefined ? val2 : undefined;

```

Comment:
Can this just collapse down to a simple sequential [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)?

```suggestion
    const val = val1 ?? val2 ?? undefined;
```

And... if it really can be simplified down this much is the method even necessary?

Reply:
Agreed, this started as a different comparison function, but we get where we get 😬

Reply:
I think the method has really become about the conditional for the wrapper class.